addons/stock/stock.py

```diff
@@ -2827,9 +2841,40 @@ def _get_prodlot_change(self, cr, uid, ids, context=None):
     }
 
     _defaults = {
-        'product_qty': 1,
+        'product_qty': 0,
     }
 
+    def _get_quants(self, cr, uid, line, context=None):
+        quant_obj = self.pool.get("stock.quant")
+        dom = [('company_id', '=', line.company_id.id), ('location_id', 'child_of', line.location_id.id), ('lot_id', '=', line.prod_lot_id.id),
+                        ('product_id','=', line.product_id.id), ('owner_id', '=', line.partner_id.id), ('package_id', '=', line.package_id.id)]
+        quants = quant_obj.search(cr, uid, dom, context=context)
+        return quants
+
+    def onchange_createline(self, cr, uid, ids, location_id=False, product_id=False, uom_id=False, package_id=False, prod_lot_id=False, partner_id=False, company_id=False, context=None):
```

Comment:
in terms of code cleaning:
- self.pool.get('x') -> self.pool['x']
- dic.update({'foo': 'bar'}) -> dic['foo'] = 'bar'